pegasus/sites.v3/code.org/views/interactive_map.haml

```diff
@@ -1,23 +1,41 @@
 %script{src: "/js/raphael.js"}
 %script{src: "/js/extended_us_map.js"}
+%link{:href=>"/css/interactive_map.css", :rel=>"stylesheet"}
+- require 'state_abbr'
 
-%div#map
-%div#infobox.greybox
+#state-list.phone-feature
+  .nav-collapse.collapse
+  .nav
+    .dropdown
+      %button.dropdown-toggle{:href => "#", "role" => "button", "data-toggle" => "dropdown", "data-target" => "#"}
+        Select Your State
+        %b.caret
+      %ul.dropdown-menu{"role" => "menu"}
+        -STATE_ABBR_WITH_DC_HASH.each do |abbr, name|
+          %a{:href => "/promote/#{abbr}"}
+            = name
+          %br
+
+%h2.tablet-feature Choose a State
+#map.tablet-feature
+#infobox.greybox.col-33
   %p.info
     Computer science drives innovation throughout the US economy, but it remains marginalized throughout K-12 education.
   %ul
     %li
       Only
-      %strong
-        =credit_count
-      states allow students to count computer science courses toward high school graduation.
-  
+      %a{:href=>'/action', style: 'color: white; cursor: pointer; text-decoration: underline', target: '_blank'}
```

Comment:
When you link it like this it also underlines the space after the text. Try something like this instead:

```
= succeed " allow students to count computer science courses toward high school graduation." do
  %a{href: '/action', style: 'color: white; cursor: pointer; text-decoration: underline', target: '_blank'}
    %strong= credit_count
    states
```